templates/services/portainer/portainer_yml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#create portainer.yml
+
+mkdir -p /srv/portainer
+{
+echo "services:"
+  echo "  portainer:"
+  echo "    image: portainer/portainer"
+  echo "    ports:"
+  echo "      - \"9000:9000\""
+  echo "    volumes:"
+  echo "      - \"/var/run/docker.sock:/var/run/docker.sock\""
+  echo "      - \"portainer_data:/data portainer/portainer\""
+  echo "version: \"2\""
```

Comment:
If you try to actually do `./cli.sh services portainer up`, you will get an error:
```
ERROR: Named volume "portainer_data:/data portainer/portainer:rw" is used in service "portainer" but no declaration was found in the volumes section.
```

This can be fixed by declaring the volume at the end of the .yml file, after `echo "version: \"2\""`:
```
echo "volumes:"
echo "  portainer_data:"
```